dockers/docker-fpm-quagga/bgpd.conf.j2

```diff
@@ -67,6 +68,10 @@ router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }}
 {% if minigraph_devices[inventory_hostname]['type'] == 'ToRRouter' %}
   neighbor {{ neighbor_addr }} allowas-in 1
 {% endif %}
+  address-family ipv4
+    neighbor {{ neighbor_addr }} activate
```

Comment:
for ipv4 neighbor, allowas-in 1 is in address-family block, for ipv6 neighbor it is outside, is this intentional?